mllib/src/main/scala/org/apache/spark/ml/r/GeneralizedLinearRegressionWrapper.scala

```diff
@@ -30,19 +30,59 @@ private[r] class GeneralizedLinearRegressionWrapper private (
   private val glm: GeneralizedLinearRegressionModel =
     pipeline.stages(1).asInstanceOf[GeneralizedLinearRegressionModel]
 
+  lazy val rFeatures: Array[String] = if (glm.getFitIntercept) {
```

Comment:
We can remove the `r` prefix and just call it `features`. The whole class is a R wrapper.